python/paddle/fluid/tests/unittests/test_sgd_op_bf16.py

```diff
@@ -158,6 +158,15 @@ def setup_params(self):
         self.grad_row_numel = 16
 
 
+@unittest.skipIf(not core.supports_bfloat16(),
```

Comment:
Asia's PR also added this but her PR did not report skipIf error? Her PR now reports skip_grad_ci not approved. Could you please actually consult her.
Because I asked wangzhen, he said he prefer do things as regulated which is passing approval requirements one by one. If this skipif is approved, it may hide other required approvals.